src/index.html

```diff
@@ -6,9 +6,57 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
+    <header class="header">
+
+      <a class="logo" href="index.html">
+        <img src="images/logo.png" alt="moyo.logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="ul">
+          <li class="li_items">
+            <a class="a_items blue-recktangle item-1  is-active" href="index.html">Apple</a>
+          </li>
```

Comment:
- ul/li_items and etc are bad class names, the class should describe the element. For example nav_list/nav__link.
- `blue-recktangle` what is this? The blue underline should be related to the active element, in your case `is-active`.
- item-1? What is the difference with other items? Get rid of such classes